components/autofill/Cargo.toml

```diff
@@ -21,7 +21,7 @@ sync15 = { path = "../sync15" }
 sync15-traits = {path = "../support/sync15-traits"}
 thiserror = "1.0"
 types = { path = "../support/types" }
-uniffi = "0.7"
+uniffi = "0.7.1"
```

Comment:
nit: I think we should use `^0.7.1` here to allow semver-compatible versions higher than the current release, it will make future uniffi version bumps less painful.